Mjml.Net.Benchmarking/TemplateBenchmarks.cs

```diff
@@ -4,25 +4,56 @@ namespace Mjml.Net.Benchmarking
 {
     public class TemplateBenchmarks
     {
-        private readonly MjmlRenderer MjmlRenderer;
-        public string TemplateAmario { get; set; }
+        private readonly MjmlRenderer MjmlRenderer;      
```

Comment:
I would write a params source here:

https://benchmarkdotnet.org/articles/features/parameterization.html#sample-introparamssource

Reply:
I'll update this.

Reply:
We'll lose the naming from the output of the benchmark. Is that what we want?

Reply:
The name from the output? You can loop over the fields and provide the file name as param and then load the actual content in the setup.